### ISSUE TITLE
[Accessibility][Functional]: On keyboard focus selected value disappears from the editable multiselect combo box.

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11195]
Split out of issue #6920

**Comment by joschect** [comment]
This has been this way for a while but I don't think we ever had specifications for the way that it is supposed to work. @betrue-final-final or @jspurlin was there ever a decision made for the way that this should be displayed?

**Comment by jspurlin** [comment]
I don't think there was ever a decision for how to handle this, but I would imagine it should behave similar to how the PeoplePicker shows multi-selected items
